app/scripts/views/mixins/external-links-mixin.js

```diff
@@ -19,7 +19,8 @@ define(function (require, exports, module) {
           var $el = $(el);
           $el
             .addClass('visible-url')
-            .attr('data-visible-url', $el.attr('href'));
+            .attr('data-visible-url', $el.attr('href'))
+            .attr('rel','noopener');
```

Comment:
Somewhere in [here](https://github.com/mozilla/fxa-content-server/blob/7eb856e45df86810ae13d81850c8d31782d90ba3/app/tests/spec/views/mixins/external-links-mixin.js#L63:L68) should work.